src/platforms/linux/daemon/dbusservice.cpp

```diff
@@ -31,6 +32,7 @@ extern "C" {
 #endif
 
 namespace {
+WireguardHelper wg_helper;
```

Comment:
This class contains only static methods. You don't  need `wg_helper`. When you need to call a method, just do: `WireguardHelper::name_of_the_method()`